src/register/components/ConfigurableRegistrationForm.jsx

```diff
@@ -207,12 +210,27 @@ const ConfigurableRegistrationForm = (props) => {
   }
 
   return (
-    <>
-      {formFieldDescriptions}
-      <div>
-        {honorCode}
```

Comment:
I initially did this, but it’s not worth it because this variable in the code is filled depending on the showConfigurableRegistrationFields state. The country field, which we hide, also depends on this parameter. So, based on these data, I decided to generate this line of text regardless of all these conditions. Also, I did not remove unused code to have minimal differences with upstream.